code/modules/mob_spawn/mob_spawn.dm

```diff
@@ -156,6 +158,11 @@
 /obj/effect/mob_spawn/ghost_role/attack_ghost(mob/user)
 	if(!SSticker.HasRoundStarted() || !loc)
 		return
+	// SKYRAT EDIT ADDITION
+	if(restricted_species && user.client?.prefs?.read_preference(/datum/preference/choiced/species) != restricted_species)
```

Comment:
I feel it should instead ask if its a type within the var... just so you can have multiple species and it isn't locked to a singular species.

Reply:
Hmm, sure